Resources/config/services.yml

```diff
@@ -103,20 +103,15 @@ services:
       calls:
           - [ setItinerariesRouter, [ '@smartesb.router.itineraries' ] ]
 
-  # PLACEHOLDERS FOR CONFIGURED HANLDERS
+  # PLACEHOLDERS FOR CONFIGURED HANDLERS
   smartesb.handlers.sync:
       class: '%smartesb.handlers.message_routing.class%'
 
   smartesb.handlers.async:
       class: '%smartesb.handlers.message_routing.class%'
 
-  smartesb.amqp.queue_manager:
-    class: Smartbox\Integration\FrameworkBundle\Components\Queues\QueueManager
-    public: false
-
   smartesb.drivers.queue.amqp:
-    class: Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\AmqpQueueDriver
-    arguments: ['@smartesb.amqp.queue_manager']
+    class: Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers\PhpAmqpLibDriver
```

Comment:
This service definition should be deleted because it is generated at compiler pass

Reply:
Done.